### PR TITLE
Improve guidance for return code usage for (robust) create.

### PR DESCRIPTION
**Pull request by tfrauenstein**
addresses issue: https://github.com/zalando/restful-api-guidelines/issues/685

### CODE REVIEW

File: chapters/http-requests.adoc [hunk 1]
```diff
@@ -90,16 +90,20 @@ implicitly creating the resource before updating
...
-resource was updated – with or without actual content returned), and {201} (if
-the resource was created)
-
-*Important:* It is good practice to prefer {POST} over {PUT} for creation of
-(at least top-level) resources. This leaves the resource identifier management under
-control of the service and not the client, and focuses {PUT} on its usage for updates.
-However, in situations where all resource attributes including the identifier
+* successful {PUT} requests return {200} or {204} (if the resource was updated - 
+with or without returning the resource), {201} (if resources have been created) 
+or {202} (if accepted and processed asynchronously).
```
Comment (tkrop) [inline chapters/http-requests.adoc:95]: I think for `PUT` with `201` this needs to stay in singular form: `(if the resource was created)`.

**Review by tkrop — COMMENTED** [review]
Only a minor comment.

File: chapters/http-requests.adoc (hunk 1 above)
Comment (tfrauenstein) [reply chapters/http-requests.adoc]: done -- pls. approve, thx.

**Comment by tkrop** [comment]
:+1:

**Comment by tfrauenstein** [comment]
👍

**Pull request merged**